doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -325,6 +325,15 @@ only as a storage format. In calculations they'll be converted to ``Float32``:
     julia> 2*Float16(4.)
     8.0f0
 
+The underscore ``_`` can be used as digit separator:
+
+.. doctest::
+
+    julia> 1_000
+    1000
+
+    julia> 10_000.10_001
```

Comment:
True!

I could also squeeze more examples with hexadecimal. etc in the same line with tuples:



```julia
julia> 10_000, 0_000.100_01, 0xdead_beef, 0b101_001
(10000,0.10001,0xdeadbeef,0x29)
```